textdb/textdb-textql/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>edu.uci.ics.textdb</groupId>
+    <artifactId>textdb</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>textdb-textql</artifactId>
+  <name>textdb-textql</name>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
jUnit needn't be added as a dependency here, as it exists in the `pom.xml` of the parent.